### PR TITLE
2 spaces, some content, black line, spacing

### PR DESCRIPTION
**Pull request by raywan**

### CODE REVIEW

File: app/index.html [hunk 1]
```diff
@@ -1,282 +1,291 @@
...
                 <div class="bounce3"></div>
+              </div>
             </div>
-        </div>
-    </div>
-    <div class="go-down">
-        <a href="#whoWeAre">
+          </div>
+          <div class="go-down">
+            <a href="#whoWeAre">
```
Comment (clemmy) [inline app/index.html]: Can you change this to `#mission`?

File: app/sass/styles.scss [hunk 2]
```diff
@@ -296,7 +296,8 @@ section {
 		}
 	}
 	&#about {
-		padding: 40px 0;
+		padding: 30vh 0;
+		height: 100vh;
```
Comment (clemmy) [inline app/sass/styles.scss:323]: Don't think we want `100vh` for the height here.

File: app/index.html [hunk 3]
```diff
@@ -1,282 +1,291 @@
...
-                    <div class="col-md-8 col-md-push-2 clearfix">
-                        <div class="section-heading scrollpoint sp-effect3">
-                            <h1>.we are bond<span>y</span>
-                            </h1>
-                            <h4>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Inventore reiciendis vel reprehenderit expedita cupiditate repellat debitis! Est qui quae consectetur temporibus a illum deleniti? Quae, adipisci praesentium quis omnis dignissimos!</h4>
-                        </div>
+                  <div class="col-md-8 col-md-push-2 clearfix">
+                    <div class="section-heading scrollpoint sp-effect3">
+                      <h1>
+                        Welcome to Zoo House
```
Comment (clemmy) [inline app/index.html]: Danny said the 
`Welcome to Zoo House - We build awesome products with awesome people` should be on top of the video.

Comment (raywan) [inline app/index.html]: What should go above the picture?

Comment (clemmy) [inline app/index.html]: the `How We Roll` content

File: app/sass/styles.scss (hunk 2 above)
Comment (raywan) [inline app/sass/styles.scss:323]: We don't want it to fill the screen?

**Comment by ntimpham** [comment]
lgtm

**Pull request merged**